### PR TITLE
[INV-46] Removed Flask-served HTML. Fixed broken router-links.

### PR DESCRIPTION
**Pull request by nskoretz**
**Added:**
- Signup.vue ~ Scaffolded the file and added a template message.
- router.ts
    - Added /signup route
    - Added redirects for /login and /logout
- Login.vue ~ Added a '$router.push' so that the page redirects upon
good /login return.

**Changed:**
- server/serverRun.py
    - endpoint /login now returns a JSON object with a message

**Removed:**
- server/serverRun.py
    - render_page()
    - protected_page()
    - signup_page()
    - login_page()
    - main_page()
- server/templates/*

### CODE REVIEW

File: server/serverRun.py [hunk 1]
```diff
@@ -261,55 +254,36 @@ def not_logged_in():
     return 'Not logged in!'
 
 
-@app.route('/signup.html', methods=['GET'])
-@logged_out()
-def signup_page():
-    return render_page('signup.html')
+# @app.route('/main.html', methods=['GET'])
```
Comment (HugoKlepsch) [inline server/serverRun.py:257]: Forget to delete this?

**Review by HugoKlepsch — COMMENTED** [review]
So clean...... 😎

File: server/serverRun.py (hunk 1 above)
Comment (HugoKlepsch) [reply server/serverRun.py:257]: Talked in mumble; keeping to possibly use later as a API endpoint.

**Review by HugoKlepsch — APPROVED** [review]

File: server/serverRun.py (hunk 1 above)
Comment (nskoretz) [reply server/serverRun.py:257]: Keeping this as it will be turned into an endpoint sometime in the near future, and this code can be recycled.

**Pull request merged**